### PR TITLE
Fix docker sorting for processed recordings

### PR DESCRIPTION
**Pull request by alejoe91**
This PR fixes the `modify_input_folder` function to make it work with preprocessed recordings.
In addition, a [full] installation on the docker side is always performed.

### CODE REVIEW

**Comment by samuelgarcia** [comment]
@alejoe91 : I think a full installation of SI is a bad idea.
If need need some specific package for some specific docker files then lets add this package in the docker itself this avoid a super big installation on the fly in the docker.

**Comment by alejoe91** [comment]
> @alejoe91 : I think a full installation of SI is a bad idea.
> If need need some specific package for some specific docker files then lets add this package in the docker itself this avoid a super big installation on the fly in the docker.

@samuelgarcia I think that for now it's fine. The packages that need to be installed also depend on the recording. E.g. if it's filtered, you will need scipy and so on...

File: spikeinterface/sorters/runsorter.py [hunk 1]
```diff
@@ -187,15 +205,21 @@ def run_sorter_docker(sorter_name, recording, docker_image, output_folder=None,
...
 
-            cmd = 'pip install --upgrade --force https://github.com/SpikeInterface/spikeinterface/archive/master.zip'
+            cmd = 'git clone https://github.com/SpikeInterface/spikeinterface.git'
+            commands.append(cmd)
+            cmd = 'cd spikeinterface'
+            commands.append(cmd)
+
+            cmd = 'pip install .[full]'
+            commands.append(cmd)
+            cmd = 'cd ..'
```
Comment (samuelgarcia) [inline spikeinterface/sorters/runsorter.py:215]: Tou should do
`pip install -e git+https://github.com/SpikeInterface/spikeinterface.git#egg=spikeinterface[full]`
it is more simpler

Comment (alejoe91) [reply spikeinterface/sorters/runsorter.py:215]: I'll try that, thanks

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**